src/test/scala/nl/knaw/dans/easy/multideposit/actions/AddFileMetadataToDepositSpec.scala

```diff
@@ -203,28 +264,48 @@ class AddFileMetadataToDepositSpec extends UnitSpec with BeforeAndAfter {
 
   it should "produce the xml for all the files" in {
     val dataset = mutable.HashMap(
-      "DATASET" -> List(datasetID, datasetID),
-      "AV_FILE" -> List("ruimtereis01/reisverslag/centaur.mpg", "")
+      "DATASET" -> List(datasetID, datasetID, datasetID),
+      "SF_DOMAIN" -> List("dans", "", ""),
+      "SF_USER" -> List("janvanmansum", "", ""),
+      "SF_COLLECTION" -> List("Jans-test-files", "", ""),
+      "SF_ACCESSIBILITY" -> List("RESTRICTED_GROUP", "", ""),
+      "AV_FILE" -> List("ruimtereis01/reisverslag/centaur.mpg", "ruimtereis01/reisverslag/centaur.mpg", "ruimtereis01/path/to/a/random/sound/chicken.mp3"),
+      "AV_FILE_TITLE" -> List("video about the centaur meteorite", "", "our daily wake up call"),
+      "AV_SUBTITLES" -> List("ruimtereis01/reisverslag/centaur.srt", "ruimtereis01/reisverslag/centaur-nederlands.srt", ""),
+      "AV_SUBTITLES_LANGUAGE" -> List("en", "nl", "")
     )
     AddFileMetadataToDeposit(1, (datasetID, dataset)).execute() shouldBe a[Success[_]]
 
-    Utility.trim(XML.loadFile(stagingFileMetadataFile(datasetID)))
-      .child
-      .map(node => (node \@ "filepath", node.child.filter(_.label == "format").head.text)) should {
-      have length 10 and
-        contain allOf(
-        ("data/ruimtereis01_verklaring.txt", "text/plain"),
-        ("data/reisverslag/deel01.docx", "application/vnd.openxmlformats-officedocument.wordprocessingml.document"),
-        ("data/reisverslag/deel01.txt", "text/plain"),
-        ("data/reisverslag/deel02.txt", "text/plain"),
-        ("data/reisverslag/deel03.txt", "text/plain"),
-        ("data/path/to/a/random/video/hubble.mpg", "video/mpeg"),
-        ("data/path/to/a/random/sound/chicken.mp3", "audio/mpeg"),
-        ("data/reisverslag/centaur.mpg", "video/mpeg"),
-        ("data/reisverslag/centaur.srt", "text/plain"),
-        ("data/reisverslag/centaur-nederlands.srt", "text/plain")
-      )
-    }
+    val expected = XML.loadFile(new File(getClass.getResource("/allfields/output/input-ruimtereis01/bag/metadata/files.xml").toURI))
```

Comment:
From the dropped [code] I was under the impression an index was used for `target={name}`.  Seeing the [content] of this file, I get the impression ingest-flow should use `<dcterms:title>` of files.xml, but is the title unique?

From this [document] I understood ingest-flow should look for the mimetype to select AV files, but it looks like I should look for `AudioObject</dcterms:type>` and  `VideoObject</dcterms:type>` in files.xml. I also matched up the `srt` files with the avFiles-without-extension, but now it looks like I should use `srt</dcterms:relation>` of the found av files.

[document]: https://docs.google.com/document/d/1qbih8nIRLbV3x1lOv_9T-jY1LRuD2ZRNDD1SBucecsc/edit
[code]: https://github.com/DANS-KNAW-jp/easy-split-multi-deposit/blob/1ed1d557525be6d49fd690d9365a5c1f7ba09fd4/src/main/scala/nl/knaw/dans/easy/multideposit/actions/CreateSpringfieldActions.scala#L70-L94
[content]: https://github.com/DANS-KNAW-jp/easy-split-multi-deposit/blob/ab49d104e63ccd0a31a8588a1a3831d3fc7594aa/src/test/resources/allfields/output/input-ruimtereis01/bag/metadata/files.xml#L6-L6